README.md

```diff
@@ -34,8 +50,8 @@ if (filesArray.truncated) {
 
 ## API
 
-### listContent.viaTreesApi(identifier, directory, token)
-### listContent.viaContentsApi(identifier, directory, token)
+### listContent.viaTreesApi({ user, repository, ref = 'HEAD', directory, token, getDetails = false })
+### listContent.viaContentsApi({ user, repository, ref = 'HEAD', directory, token })
```

Comment:
No need to repeat the options here, just use `options`

```md
### listContent.viaTreesApi(options)
### listContent.viaContentsApi(options)
```

Also both APIs should have the same options

Reply:
So that `getDetails` in the `viaContentsApi` would yield the GitHub API results from the contents API? And I attach a hint to the option's description that the details provided by `viaContentsApi` and `viaTreesApi` are different?

Reply:
Yes. The methods make it clear which API they use and this flag should mean "get the whole object received by the API"

Reply:
Alright, sounds reasonable. `getDetails` as a name is fine? Other suggestions?

Reply:
Either that or `getFullData`, perhaps